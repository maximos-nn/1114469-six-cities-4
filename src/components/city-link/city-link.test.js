import React from "react";
import renderer from "react-test-renderer";
import CityLink from "./city-link.jsx";

it(`CityLink component should render correctly`, () => {
  const tree = renderer.create(
      <CityLink active={false} city={`Amsterdam`} onClick={() => {}} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it(`Active CityLink component should render correctly`, () => {
  const tree = renderer.create(
      <CityLink active={true} city={`Amsterdam`} onClick={() => {}} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

