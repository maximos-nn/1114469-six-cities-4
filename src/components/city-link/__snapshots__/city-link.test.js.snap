// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Active CityLink component should render correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item  tabs__item--active"
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;

exports[`CityLink component should render correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item"
    href="#"
    onClick={[Function]}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
